package.json

```diff
@@ -65,6 +65,9 @@
   "preferGlobal": false,
   "c8": {
     "all": true,
+    "src": [
+      "add-on/src/"
+    ],
```

Comment:
This seems to fix CI hangs,  i think c8 was processing `./` which includes `node_modules` or :shrug: